UpdateProfile/handler.ts

```diff
@@ -103,11 +103,27 @@ export function UpdateProfileHandler(
       emailChanged ? false : existingProfile.isEmailValidated
     );
 
+    const autoEnableInboxAndWebHook =
```

Comment:
are we sure this is the condition? (can it be from acceptedTosVersion = 0 to acceptedTosVersion = 1 instead of undefined ?) cc @francescopersico 

in this case we could use `!existingProfile.acceptedTosVersion && profile.acceptedTosVersion`
which includes `0` and `undefined`

Reply:
@gunzip the API `createProfile` doesn't get in input `acceptedTosVersion` so the initial value should be `undefined`

Reply:
@Undermaken how the app accept the ToS after the login?

Reply:
Yes it does.

- login
- app gets the profile
- if `profile.acceptedTosVersion` is `undefined` or `profile.acceptedTosVersion < HARD_CODED_VALUE_IN_APP`
- app shows the ToS screen
- user accepts
- app sends a` /profile POST` where `acceptedTosVersion = HARD_CODED_VALUE_IN_APP`